### DISCUSSION TITLE
One Side ADF Sorting

### DISCUSSION THREAD

**Discussion by 321jaber** [cyanfish/naps2#292] [General]
Hi!
Is it possible to include function to use single side adf as duplex ADF?
It is almost as batch scaning, but it need to sort second batch backwards...

So if i have 5 double side pages first bach is scaning odd sides
1 3 5 7 9 (front sides of pages)
And the second is scanning even sides but backwards :-)
10 8 6 4 2

so in final we have 
1 3 5 7 9 10 8 6 4 2 

It would be grate if program could sort it correctly...
Thanks a lot!

**Comment by cyanfish** [comment]
Click Reorder->Alternative Interleave to do that.

**Reply by 321jaber** [reply]
Thanks!
Its amazing :-)
